CompanionPane/src/main/res/layout-land/dual_screen_layout_end.xml

```diff
@@ -4,60 +4,54 @@
   ~  Licensed under the MIT License.
   ~
   -->
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/dual_screen_layout_land_end"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:baselineAligned="false"
-    android:orientation="vertical"
     android:background="@drawable/background_portrait_end"
+    android:padding="@dimen/large_padding"
     tools:ignore="MissingDefaultResource">
 
-    <androidx.appcompat.widget.Toolbar
-        android:id="@+id/toolbar"
+    <include
+        android:id="@+id/magic_definition_control"
+        layout="@layout/magic_definition_control"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:background="@color/white"
-        android:minHeight="?attr/actionBarSize"
-        android:layout_marginBottom="@dimen/large_padding" />
+        android:layout_gravity="center"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
 
-    <LinearLayout
```

Comment:
I like the flattening of the hierarchy, but can we make some changes to keep the center position of the controls? I feel like it looks a bit off with so much space at the bottom

Reply:
Done